jobs/signing/sign-artifacts/Jenkinsfile

```diff
@@ -206,12 +205,16 @@ node {
                 //
                 // where <RELEASE-NAME> is something like '4.1.0-rc.5'
 
-                // transform the file into a directory containing the file
+                // transform the file into a directory containing the
+                // file, mirror to google
                 commonlib.shell('''
                     for file in sha256=*; do
-                        mv $file signature-1
+                        mv $file ${params.SIGNATURE_NAME}
```

Comment:
won't work in a block with `'''`
needs to be `"""` and then backslash-escape `$file` everywhere

Reply:
actually i suppose it won't care about `$file` as long as it's not in braces